### DISCUSSION TITLE
Cannot return to the home screen directly

### DISCUSSION THREAD

**Discussion by lanyizi** [community/community#4101] [Mobile] [answered]
I have to go back and go back until I'm eventually back into the home screen, can we add some kind of "shortcut"?

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to go to Home internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Reply by maverick1872** [reply]
Is there any update on this? It's a glaring usability issue and can make keeping tabs on things exceptionally tedious.

**Reply by aconsuegra** [reply]
Hi @maverick1872. It's still in our roadmap, but we don't have an estimation about when it could land in the apps yet.

**Comment by raedfah** [comment]
Hello

**Comment by raedfah** [comment]
I Need help

**Comment by CanRau** [comment]
Yes please, would prefer to have the bottom nav bar always visible instead of having to go back 10 times 😵‍💫
